documentation/cluster-controller/cluster-controller.adoc

```diff
@@ -0,0 +1,267 @@
+= Cluster Controller
+
+The cluster controller is in charge to deploy a Kafka cluster alongside a Zookeeper ensemble. It's also able to deploy a 
+Kafka Connect cluster which connects to an already existing Kafka cluster.
+
+When up and running, it starts to "watch" for ConfigMaps containing the Kafka or Kafka Connect cluster configuration.
```

Comment:
You talked about Kafka, ZK and KAfka Connect before ... it might make sense to clarify that you mean "when the Cluster Controller is up and running".